### DISCUSSION TITLE
ItemsAdder support

### DISCUSSION THREAD

**Discussion by ItzAndriss** [OmerBenGera/WildStacker#831] [Pending Feature Requests]
ItemsAdder items should not be named feather, but should be set individually.
![2023-08-22_13 11 33](https://github.com/BG-Software-LLC/WildStacker/assets/81358652/d8321adf-9eb4-4c4b-9fb9-03a9820bf05e)

**Comment by OmerBenGera** [comment]
> ItemsAdder items should not be named feather, but should be set individually. ![2023-08-22_13 11 33](https://user-images.githubusercontent.com/81358652/262332502-d8321adf-9eb4-4c4b-9fb9-03a9820bf05e.png)

You can have the names of the items be displayed instead of their type - this is a configurable option.

**Reply by ItzAndriss** [reply]
Thank you very much for your response, I managed to solve this.
